components/playground/main.go

```diff
@@ -82,14 +85,16 @@ if you don't specified a version.
 Examples:
   $ tiup playground nightly                         # Start a TiDB nightly version local cluster
   $ tiup playground v3.0.10 --db 3 --pd 3 --kv 3    # Start a local cluster with 10 nodes
-  $ tiup playground nightly --monitor               # Start a local cluster with monitor system`,
+  $ tiup playground nightly --monitor               # Start a local cluster with monitor system
+  $ tiup playground --db.binpath /xx/tidb-server --pd.binpath /xx/pd-server --kv.binpath /xx/tikv-server
```

Comment:
```suggestion
  $ tiup playground --db.binpath /xx/tidb-server
```
I think we don't need to demo all flags in the examples, the users can learn from `tiup help playground`.